example/android/build.gradle

```diff
@@ -2,17 +2,18 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "28.0.3"
+        buildToolsVersion = "30.0.2"
         minSdkVersion = 21
-        compileSdkVersion = 28
-        targetSdkVersion = 28
+        compileSdkVersion = 31
```

Comment:
need to lower compile+target sdk to 30 as most customers are not on 31 yet

Reply:
done